Lesson13/Kadir/conditionalStatements.js

```diff
@@ -89,14 +172,101 @@ election2(14);
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+let height=parseInt(prompt("Input your height(m)= "));
+let weight=parseInt(prompt("Input your weight(kg)= "));
+
+function indexBMI(h, w){
+  if (w/(h^2)<=18.5){
+    console.log("You are underweight. You need to eat much more..");
+  } else if (w/(h^2)>18.5 && w/(h^2)<=24.9){
+    console.log(" You have Healthy Weight. So go on what you are doing..");
+  } else if (w/(h^2)>24.9 && w/(h^2)<=29.9){
+    console.log(" You are overweight. Get an action soon..");
+  } else {
+    console.log(" You have obesity. Go to a doctor..");
+  }
+}
+
+indexBMI(height, weight);
+
+
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
 
+let age=parseInt(prompt("Input your age= "));
+
+function checkAgeGroup(num){
+  if (num<=12){
+    console.log(num +"You are a child.");
+  } else if (num<=19){
+    console.log(num +"You are a teenager.");
+  } else if (num<=64) {
+    console.log(num +"You are an adult.");
+  } else {
+    console.log(num +"You are a senior citizen.");
+  }
+}
+
+checkAgeGroup(age);
+
 // 10. Write a program that asks the user to enter their age and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
 
+let age2=parseInt(prompt("Input your age: "));
+let student=prompt("Are you a student (Y/N): ");
+
+function canVote(a, b){
```

Comment:
```suggestion
function canVote(age, isStudent){
```

Reply:
For the isStudent parameter, use boolean type (true/false) then your code will be cleaner and a lot less error-prone.
E.g. `if (age<18 && isStudent){`